### DISCUSSION TITLE
(Attempting to) sort searches by relevance

### DISCUSSION THREAD

**Discussion by ilc168** [LesFerch/WinSetView#114] [Q&A] [answered]
Hello, windows 11 user. I've set all my search results to sort by relevance, and set relevance to display as a column heading. However, searches still appear sorted by name in searches. And there is no relevance field in searches (that's ok, I'm more interested in sorting via relevance).

It says it is sorted by relevance in the menu item as per image below. Am I missing something?

Thanks

![image](https://github.com/LesFerch/WinSetView/assets/20764716/54b4ebb9-f902-41a4-ad6c-8f82b0b4890a)

**Comment by LesFerch** [comment, marked as answer]
It should work. I tested the same settings and I got a Relevance column heading in Search Results and the results were ordered by Relevance:

![image](https://github.com/LesFerch/WinSetView/assets/79026235/99dfc5e5-13ff-43e8-94b2-5dec4cd181fb)

Of course, the folder must be indexed in order to get a relevance score, but even for non-indexed locations the Relevance column appears in the Search Results with a score of 0 for all items.

Did you click **Submit** after making those selections in WinSetView?

**Reply by ilc168** [reply]
Thanks for the quick response. I had clicked submit. 

I have the relevance column showing now. I don't know if it was checking/unchecking 'relevance' in windows options or re-doing in WinSetView that did it. 

I cleared and updated indexing just in case that was an issue for sorting via relevance. 

I think the sub-par ordering may be a function of windows search tool. My downloads folder all have equal relevance when I search for an obvious term (e.g. word in document title). So the default sort is then to alphabetical as all the relevance values were the same.

**Reply by LesFerch** [reply]
My sense is that an indexed folder full of Word documents or text files may result in a greater variety of relevance scores as the contents would be searched as well. When only filenames are involved, it's pretty hard to even get three different relevance scores.
